### ISSUE TITLE
Printing causes the process to crash

### ISSUE THREAD

**Issue by HyfMln** [electron/electron#15188]
When calling <webview>.print()  or window.print(),All processes crash.

**Comment by MarshallOfSound** [comment]
It looks like you didn't fill out our [Issue Template](https://raw.githubusercontent.com/electron/electron/master/.github/ISSUE_TEMPLATE.md), please ensure you provide all information requested in the issue template so that we can try to help you out.

In the future issues you create without the template will be closed, you can find the issue template [here](https://raw.githubusercontent.com/electron/electron/master/.github/ISSUE_TEMPLATE.md) and it is automatically populated in the new issue form.

**Comment by damianaiamad** [comment]
Electron 4.0.0-beta.4 
Windows 10

**Expected Behaviour**
Electron does not crash after printing.

**Actual Behaviour** 
Printing works but electron crashes immediately after.

May be related to [window.print() only works once #14705](https://github.com/electron/electron/issues/14705)

**To Reproduce**
[Repo Link](https://github.com/damianaiamad/print-test)

Run npm install
Run .\node_modules\electron\dist\electron.exe .
Clicking on the document will trigger a window.print()

**Comment by jkleinsc** [comment]
I can reproduce this issue on Electron 4.0.0-beta.4 and Electron 4.0.0-beta.6

Electron Fiddle repro here:
https://gist.github.com/bf55ecc157b5e4d4a69a0a07e0ac0610

It looks like this was likely caused by #15023.

#15143 may resolve this issue.

cc @brenca @deepak1556

**Comment by brenca** [comment]
@jkleinsc #15023 didn't make it to the `4-0-x` branch yet AFAIK, and it would make sense to do the actual backport once the followup PR lands.

**Comment by jkleinsc** [comment]
@brenca my bad.  yes that change didn't land in 4-0-x.  Is the followup PR @deepak1556's PR?

**Comment by deepak1556** [comment]
@jkleinsc yup https://github.com/electron/electron/pull/15143 is the followup, still a few cleanups left, I will make sure to backport it once merged.

**Comment by imanshugupta** [comment]
I am also facing this issue with electron
my application crashes whenever i call window.print() either in the console window or in the application itself
